libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -62,15 +62,10 @@ export class StripeManager {
    * Creating a subscription with automatic_tax enabled requires a customer with an address
    * that is in a recognized location with an active tax registration.
    */
-  async isCustomerStripeTaxEligible(customer: Stripe.Customer) {
-    if (!customer.tax) {
-      // TODO: FXA-8891
-      throw new Error('customer.tax is not present');
-    }
```

Comment:
Just want to double check this. Is it correct to say that `Stripe.Customer.Tax` is only possibly `undefined`, because it isn't always expanded on a `Stripe.Customer`? And since with `StripeCustomer`, `tax` is always expanded, it will always have value?

Reply:
I realized this comment was a bit vague. Put another way, I just wanted to make sure it's valid to say that `StripeCustomer.tax` will always have a value, and won't be undefined?

Reply:
So there’s a few rare cases where there’s no way to fetch some specific data on an object without using expand. Stripe.Customer’s tax property is one of those - there’s no other endpoint one can fetch it.

Fortunately for the purposes of “we don’t want to use expand”, I’ve implemented the types in such a way to where it actually doesn’t matter whether we use expand or not - just so long as we always use expand or we never use expand for a given property on a given type.

In this case, I've guaranteed that `tax` will always be expanded on a customer